src/app/components/Text/README.md

```diff
@@ -0,0 +1,13 @@
+# Text [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind=Text)
+
+## Description
+
+The 'Text' component is intended to be used to parse all article story text. It takes [markdown](https://en.wikipedia.org/wiki/Markdown) as an input and converts it to the desired HTML markup. It has special treatment for inline links as they are handled differently if they point to a link inside Simorgh.
```

Comment:
```suggestion
The 'Text' component is being used to parse all article story text. It takes [markdown](https://en.wikipedia.org/wiki/Markdown) as an input and converts it to the desired HTML markup. 
```

`intended` makes it sounds like this is it's only use case, not one use case of many. 

The text component isn't worried about the Simorgh inline link logic, the SPA link logic comes from the Inline Link Container and therefore shouldn't have affect on the Text component as a reusable React component

Reply:
You could mention that the styling of the component can be overridden by those components (EG: the Inline Link component)